packages/storybook/stories/components/icon/icon.stories.tsx

```diff
@@ -288,3 +288,43 @@ export const Informative: Story = {
       name="circle-question" />
   ),
 };
+
+export const AccessibilityBadPracticeDecorative: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <Icon
+      name="home" />
+  ),
+};
+
+export const AccessibilityBadPracticeRating: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <>
+      <label htmlFor={ 'rating' }>Rating</label>
+      <div id={ 'rating' }>
+        <Icon name="star" tabIndex={ 0 } aria-label="one star" />
+        <Icon name="star" tabIndex={ 0 } aria-label="two star" />
+        <Icon name="star" tabIndex={ 0 } aria-label="three star" />
+        <Icon name="star" tabIndex={ 0 } aria-label="four star" />
+        <Icon name="star" tabIndex={ 0 } aria-label="five star" />
+      </div>
+    </>
+  ),
+};
+
+export const AccessibilityRating: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <>
+      <label id="rating-label" htmlFor={ 'rating' }>Rating</label>
```

Comment:
you can use directly `=""` when setting string instead of `{ '' }`
(on this line you're using both way)